frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.pdf.component.js

```diff
@@ -1,19 +1,625 @@
+          top="43.3%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.creditCardPayments"
+          top="45.5%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.loanAndOtherDebt"
+          top="47.7%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.alimony"
+          top="50%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.childSupport"
+          top="52.5%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.childCare"
+          top="54.7%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.extraActivitiesChildren"
+          top="57%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.educationChild"
+          top="59.4%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.educationSelf"
+          top="61.7%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.healthInsurance"
+          top="64%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.healthExpenses"
+          top="66.3%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.otherInsurance"
+          top="68.6%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.entertainment"
+          top="70.9%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.laundryDryClean"
+          top="73.2%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.donations"
+          top="75.5%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.gifts"
+          top="77.9%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.unionDues"
+          top="80.2%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.garnishments"
+          top="82.6%"
+          left="69.5%"
+        />
+        <PositionedString
+          dataKey="person.expenses.retireDeposits"
+          top="84.8%"
+          left="69.5%"
+        />
+      </RenderPage>
+      <RenderPage url="/static/forms/motion-to-waive-fees/Motion_to_Waive_Fees_and_Statement_Supporting-06.png">
+        <PositionedCheckmark
+          dataKey="person.businessInterest"
+          top="28%"
+          left="18%"
+        />
+        <PositionedCheckmark
+          dataKey="person.businessInterest"
+          top="25.7%"
+          left="18%"
+          shouldShow={data.person.businessInterest != true}
```

Comment:
generally speaking in javascript, it's best to always do a `===` or `!==` check instead of `==` and `!=`.

The reason is that triple equals does not do [type coercion](https://developer.mozilla.org/en-US/docs/Glossary/Type_coercion), but double equals does.

Type coercion is notoriously complex and confusing, so the javascript community usually shuns it. Let's update all of the `!=` and `==` to `!==` and `===`